lua/nvim-treesitter/configs.lua

```diff
@@ -22,7 +22,8 @@ local builtin_modules = {
     enable = false,
     disable = {'markdown'}, -- FIXME(vigoux): markdown highlighting breaks everything for now
     custom_captures = {},
-    is_supported = queries.has_highlights
+    is_supported = queries.has_highlights,
+    additional_vim_regex_highlighting = false,
```

Comment:
Maybe have this true only for some languages?

Reply:
This will now allow either `true` or `{ 'python', 'cpp' }` if you only want to have this for some languages.